index.html

```diff
@@ -27,9 +27,9 @@ <h1>Language Server Protocol</h1>
         <div class="col-lg-7">
             <h2 class="header-light regular-pad">What is the Language Server Protocol?</h2>
 
-            <p>Implementing support for features like autocomplete, goto definition, or documentation on hover for a programming language is a significant effort. Traditionally this work must be repeated for each development tool, as each provides different APIs for implementing the same features.</p>
-            <p>The idea behind a <i>Language Server</i> is to provide the language-specific smarts inside a server that can communicate with development tooling over a protocol that enables inter-process communication.</p>
-            <p>The idea behind the <i>Language Server Protocol (LSP)</i> is to standardize the protocol for how tools and servers communicate, so a single <i>Language Server</i> can be re-used in multiple development tools, and tools can support languages with minimal effort.</p>
+            <p>Adding features like auto complete, go to definition, or documentation on hover for a programming language takes significant effort. Traditionally this work had to be repeated for each development tool, as each tool provides different APIs for implementing the same feature.</p>
```

Comment:
FWIW "autocomplete" seems to be used with [higher frequency](https://books.google.com/ngrams/graph?content=autocomplete%2C+auto+complete&year_start=1800&year_end=2000&corpus=15&smoothing=3&share&&direct_url=t1%3B%2Cautocomplete%3B%2Cc0%3B.t1%3B%2Cauto%20complete%3B%2Cc0) than "auto complete"

Reply:
Am ok with either, as long as we are consistent